### DISCUSSION TITLE
More customizeability in Hold-to-confirm Activation Time

### DISCUSSION THREAD

**Discussion by dinkelspiel** [ppy/osu#18055] [Ideas] [answered]
I really like to have the Hold-to-Confirm time at 0ms when playing in singleplayer as i'm used to in stable but when playing multi it would be nice to have a seperate Hold-to-Confirm time as currently if i have the hold-to-confirm time at 0ms and press escape in a multi i instantly leave the game which gets quite frusturating so either a seperate hold-to-confirm time or a pause menu in the multi would be a nice feature.

Best Regards,
Willem

**Comment by peppy** [comment, marked as answer]
I think leaving multiplayer should be classified as a dangerous action, foricng a hold at the same fixed delay as `DangerousConfirmContainer`.

**Reply by peppy** [reply]
Preliminary effort towards this at #18069.

**Reply by bdach** [reply]
Marking as answer as the linked PR is inevitably heading for merge.
